src/commands/force/source/deploy.ts

```diff
@@ -227,15 +237,27 @@ export class Deploy extends DeployCommand {
   }
 
   protected formatResult(): DeployCommandResult | DeployCommandAsyncResult {
-    const formatterOptions = {
+    this.flags.outputdir = this.resolveOutputDir(
```

Comment:
Is it a good idea to overwrite flags ? I guess you can. but I think these objects should be immutable...

Reply:
I'll look at changing this.